letsencrypt/plugins/common.py

```diff
@@ -90,6 +90,16 @@ def add_parser_arguments(cls, add):
 
         """
 
+    @property
```

Comment:
Why make a property of such uncomplicated thing? Just say `is_root_required = True`